clickhouse-data/src/main/java/com/clickhouse/data/format/BinaryStreamUtils.java

```diff
@@ -224,6 +224,18 @@ public static void writeBytes(OutputStream output, byte[] bytes) throws IOExcept
         output.write(bytes);
     }
 
+    /**
+     * Writes string bytes into given output stream.
+     *
+     * @param output non-null output stream
+     * @param string  non-null utf-8 string
+     * @throws IOException when failed to write value to output stream or reached
+     *                     end of the stream
+     */
+    public static void writeBytes(OutputStream output, String string) throws IOException {
+        output.write(string.getBytes(StandardCharsets.UTF_8));
```

Comment:
Is it correct that method doesn't write a length before bytes? 
Also please take a look at `com.clickhouse.data.format.BinaryStreamUtils#writeString(java.io.OutputStream, java.lang.String, java.nio.charset.Charset)` - it does similar things. Will it work for your case?